### PR TITLE
Update config-template.yml

### PR DESCRIPTION
**Pull request by scality-gelbart**

### CODE REVIEW

**Comment by scality-trevorbenson** [comment]
@scality-gelbart Have you run a complete set of `s3_fsck_p0.py` through `s3_fsck_p4.py` and confirmed there are zero errors from this change?

I am hesitant because I had to resolve double slashes and use os.path.join for some parts of code. Specifically the file vs. s3a protocols. Filesystems don't care, but S3 (s3a pyspark module at least) does not handle well "double prefix delimiters". Meaning when `s3://bucketname/prefix1/prefix2/` contains files the path `s3://bucketname/prefix1//prefix2/` does not contain files. I'm pretty confident URL's should be handled like Filesystems in most cases, but want to confirm this is well tested before merging.

**Review by scality-trevorbenson — COMMENTED** [review]
Looks good. Needs at least testing against every `s3_fsck` script at a minimum. If that succeeds we can merge it and presume there will be no fallout from the SOFS or RING Orphans scripts  as I don't have someone who has used them recently enough to request testing from.

Thanks

**Review by scality-trevorbenson — CHANGES_REQUESTED** [review]
@scality-gelbart I haven't heard back and was thinking of merging. However looking over the current code I'd like to see the following patches applied to your branch before approval. This should ensure that if the http client (or server) handling of double slashes changes that it does not introduce an error due to the double forward slash after the port number which would otherwise be produced: `http:/127.0.0.1:81//rebuild/[...]`

```
diff --git a/scripts/S3_FSCK/s3_fsck_p0.py b/scripts/S3_FSCK/s3_fsck_p0.py
index 432fc6c..7c82672 100644
--- a/scripts/S3_FSCK/s3_fsck_p0.py
+++ b/scripts/S3_FSCK/s3_fsck_p0.py
@@ -118,7 +118,7 @@ def sparse(f):
 
 
 def check_split(key):
-    url = "http://%s:81/%s/%s" % (SREBUILDD_IP, SREBUILDD_ARC_PATH, str(key.zfill(40)))
+    url = "http://%s:81%s/%s" % (SREBUILDD_IP, SREBUILDD_ARC_PATH, str(key.zfill(40)))
     r = requests.head(url)
     if r.status_code == 200:
         split = r.headers.get("X-Scal-Attr-Is-Split", False)
diff --git a/scripts/S3_FSCK/s3_fsck_p3.py b/scripts/S3_FSCK/s3_fsck_p3.py
index 6243e85..853d73f 100644
--- a/scripts/S3_FSCK/s3_fsck_p3.py
+++ b/scripts/S3_FSCK/s3_fsck_p3.py
@@ -25,7 +25,7 @@ SECRET_KEY = cfg["s3"]["secret_key"]
 ENDPOINT_URL = cfg["s3"]["endpoint"]
 SREBUILDD_IP  = cfg["srebuildd_ip"]
 SREBUILDD_ARCDATA_PATH  = cfg["srebuildd_arcdata_path"]
-SREBUILDD_URL = "http://%s:81/%s" % (SREBUILDD_IP, SREBUILDD_ARCDATA_PATH)
+SREBUILDD_URL = "http://%s:81%s" % (SREBUILDD_IP, SREBUILDD_ARCDATA_PATH)
 ARC = cfg["arc_protection"]
 COS = cfg["cos_protection"]
 
diff --git a/scripts/S3_FSCK/s3_fsck_p4.py b/scripts/S3_FSCK/s3_fsck_p4.py
index b7ffb0d..ef21a7b 100644
--- a/scripts/S3_FSCK/s3_fsck_p4.py
+++ b/scripts/S3_FSCK/s3_fsck_p4.py
@@ -31,7 +31,7 @@ URL = cfg["sup"]["url"]
 PATH = cfg["path"]
 SREBUILDD_IP  = cfg["srebuildd_ip"]
 SREBUILDD_PATH  = cfg["srebuildd_chord_path"]
-SREBUILDD_URL = "http://%s:81/%s" % (SREBUILDD_IP, SREBUILDD_PATH)
+SREBUILDD_URL = "http://%s:81%s" % (SREBUILDD_IP, SREBUILDD_PATH)
 PROTOCOL = cfg["protocol"]
 ACCESS_KEY = cfg["s3"]["access_key"]
 SECRET_KEY = cfg["s3"]["secret_key"]
```
